python/python/glide/async_commands/cluster_commands.py

```diff
@@ -415,6 +416,54 @@ async def sort(
         result = await self._execute_command(RequestType.Sort, args)
         return cast(List[str], result)
 
+    async def sort_ro(
+        self,
+        key: str,
+        limit: Optional[Limit] = None,
+        order: Optional[OrderBy] = None,
+        alpha: Optional[bool] = None,
+    ) -> List[str]:
+        """
+        Sorts the elements in the list, set, or sorted set at `key` and returns the result.
+        This command is routed depending on the client's `ReadFrom` strategy.
```

Comment:
Probably this should be copied to standalone client too - it can a replica as well.